semantic/src/site/globals/site.overrides

```diff
@@ -123,6 +123,38 @@ p strong {
   margin-left: 10px;
 }
 
+/* Application home */
+#application-area {
+  font-family: @userAreaFont;
+  display: flex;
+  flex-direction: column;
+}
+
+.company-title .h4 {
```

Comment:
You probably mean `h4` not `.h4`, right?
But why have the h4 tag at all?

Reply:
Just my attempt to change the h4 definition. Will do as you suggested in another comment

Reply:
Done: a58df28